CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+## How to contribute
+
+The purpose of this page is to provide you with the minimum valuable information about how to contribute to the
+project.
+
+### How reach us 🗞
+
+- you can join the slack community [here](https://androiddevs.it/)
+- and follow the open [discussion on the repository](https://github.com/androiddevelopersitalia/aditogether/discussions)
+
+### Set up the project ⚙️
+
+- Have a JDK >= 11 installed (even the bundled one with AS/IJ)
+- Clone the repository
+- Run `./scripts/tuner.sh` to set up the local tools used by this repository
+
+### Branch & commits convention 📙
+
+We have decided to use the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/), this means that there
+is a specific naming convention for branch and about how to write the commits message.
+
+For instance if you are working on a new feature the branch name must have the `feat/` prefix and then a reference about
+the feature:
+
+```
+feat/1-adding-profile-section
+```
+
+The same thing for the commit message:
+
+```shell
+git commit -m "feat(profile): implemented profile section"
```

Comment:
See my global comment, I think it also fits here.

Reply:
+1 on imperative form for commits